vultr/vultr.py

```diff
@@ -686,9 +686,9 @@ def server_reboot(self):
             found using the v1/server/list call.
         """
         params = {'SUBID': subid}
-        return self.request('/v1/server/reboot', params, 'POST')
+        return self.request('/v1/server/reboot', params, 'POST', json=False)
```

Comment:
The default is True.  In this particular case we set it to False because this API point always returns the empty string, which breaks JSON.  I haven't checked whether there is any API point that returns a non-JSON, non-empty string.  If there is none, you can probably simplify this by removing the `json` parameter and just checking for the empty string right before returning from `request`.